### PR TITLE
[#27] Add PR & issues templates for GitHub

### LINKED ISSUE #27: Add the PR template for React projects
## Why

When creating the application from the template, there are no ready-to-use PR templates:

image

However, the PR template is required as per [our team conventions](https://nimblehq.co/compass/development/code-reviews/#opening-a-pull-request).
 
## Who Benefits?

Nimble Developers

### PR DESCRIPTION
**Pull request by malparty**
Should close #27 

## What happened 

- [x] Add the `.github` folder inside the `template` folder (including PR and issues templates)
- [x] Tested

Extra
- Add the contribution explanations, as I could not remember how to use the template from a local version

## Insight

`N/A`

## Proof Of Work 💪

Tested on the freshly new deployed template:

image

### CODE REVIEW

**Comment by carryall** [comment]
Hi @malparty, I just recognized that we have [an issue template for `chore` tasks](https://github.com/nimblehq/github-actions-workflows/blob/develop/.github/ISSUE_TEMPLATE/chore_template.md) so how about adding one here?
The template content can be the same as the feature templates except the label changed to `chore` 🙌🏼 

`labels: "type : feature"`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-## What happened 
+## What happened 👀
 
 Describe the big picture of your changes here to communicate to the team why we should accept this pull request.
 
-## Insight
+## Insight 📝
 
-Describe in details how to test the changes. Referenced documentation are welcome as well.
+Describe in detail how to test the changes. Referenced documentation is welcome as well.
```
Comment (hoangmirs) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: As you made some changes to the git templates, could you also open a PR to fix them on [this repository](https://github.com/nimblehq/git-template/) as well?

**Review by hoangmirs — APPROVED** [review]
LGTM

**Comment by malparty** [comment]
> Hi @malparty, I just recognized that we have [an issue template for `chore` tasks](https://github.com/nimblehq/github-actions-workflows/blob/develop/.github/ISSUE_TEMPLATE/chore_template.md) so how about adding one here? The template content can be the same as the feature templates except the label changed to `chore` 🙌🏼
> 
> `labels: "type : feature"`
> 
> And maybe we should also add the [template for release pull request](https://github.com/nimblehq/github-actions-workflows/blob/develop/.github/PULL_REQUEST_TEMPLATE/release_template.md) 💭

@carryall done in 2ce0bab 🚀

**Comment by malparty** [comment]
6aee328 adds the `.github` folder into the generated app from the template. As this is pretty similar ;-)

**Comment by malparty** [comment]
@carryall PR added in the GitHub Action Workflow repo: https://github.com/nimblehq/github-actions-workflows/pull/5

**Comment by carryall** [comment]
> > Hi @malparty, I just recognized that we have [an issue template for `chore` tasks](https://github.com/nimblehq/github-actions-workflows/blob/develop/.github/ISSUE_TEMPLATE/chore_template.md) so how about adding one here? The template content can be the same as the feature templates except the label changed to `chore` 🙌🏼
> > `labels: "type : feature"`
> > And maybe we should also add the [template for release pull request](https://github.com/nimblehq/github-actions-workflows/blob/develop/.github/PULL_REQUEST_TEMPLATE/release_template.md) 💭
> 
> @carryall done in [2ce0bab](https://github.com/nimblehq/react-templates/commit/2ce0bab19c63b30b79a898ff62f1df681fa5de41) 🚀

Hi, @malparty I think that commit goes in the wrong branch as it is shown in [this PR](https://github.com/nimblehq/react-templates/pull/41) 😂

**Review by carryall — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (malparty) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: @hoangmirs I give you not 1 but 2 PR :D 
A PR in the [git-template](https://github.com/nimblehq/git-template/pull/20) repo.
Another PR in the [github-actions-workflows](https://github.com/nimblehq/github-actions-workflows/pull/5) repo. ;-)

**Pull request merged**